src/corehost/cli/fxr/sdk_info.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef __SDK_INFO_H_
+#define __SDK_INFO_H_
+
+#include "libhost.h"
+
+struct sdk_info
+{
+    sdk_info(pal::string_t path, fx_ver_t version)
+        : path(path)
+        , version(version) { }
+
+    static void get_all_sdk_infos(
```

Comment:
Since this is a lightweight utility class, I didn't see a reason to make things private. Some code in the future may want to construct one or access the fields.

For the SDK class, yes the ctor and two fields could be private.

For the Framework class, the ctor could be private, but the fields must be public since they are used by code in fx_muxer.cpp.